q2_fmt/_peds.py

```diff
@@ -253,15 +274,15 @@ def _compute_peds(peds_df: pd.Series, peds_type: str, peds_time: int,
             'title': subject_column,
             'description': 'Subject IDs linking samples across time'
         })
-        peds_df["transfered_donor_features"].attrs.update({
-            'title': "Transfered Donor Features",
+        peds_df[transfered].attrs.update({
+            'title': "Transfered reference Features",
```

Comment:
the capitalization on these titles is also inconsistent - maybe pick either first word capitalized or all words capitalized, just so it looks cleaner to the user?